File: blog-sever/server-java/src/main/java/com/site/blog/model/dto/BlogPageCondition.java
package com.site.blog.model.dto;

import lombok.Data;
import lombok.experimental.Accessors;


@Data
@Accessors(chain = true)
public class BlogPageCondition {

    private Integer pageNum;

    private Integer pageSize;

    private String keyword;

    private String pageName;

    private String tagId;

    private String categoryId;

}

File: myblog-kt/src/main/kotlin/com/site/blog/model/dto/BlogPageCondition.kt
package com.site.blog.model.dto

data class BlogPageCondition (
    var pageNum: Int? = 0,
    var pageSize: Int? = 20,
    var keyword: String? = null,
    var pageName: String? = null,
    var tagId: String? = null,
    var categoryId: String? = null,
)